.vscode/launch.json

```diff
@@ -2,6 +2,30 @@
     "version": "0.2.0",
     "configurations": [
         {
+            "name": "Test water level model (dataset)",
+            "type": "debugpy",
+            "request": "launch",
+            "program": "deep_water_level/src/infer.py",
+            "console": "integratedTerminal",
+            "args": [
+                "--crop_box", "130", "275", "140", "140",
+                "--model_path", "model.pth",
+                "--dataset_dir", "datasets/water_test_set3",
+                "--annotations_file", "filtered.json",
+            ]
+        },
+        {
+            "name": "Test water level model (single image)",
+            "type": "debugpy",
+            "request": "launch",
+            "program": "deep_water_level/src/infer.py",
+            "console": "integratedTerminal",
+            "args": [
+                "--crop_box", "130", "275", "140", "140",
```

Comment:
I wonder if there is an easy way to store extra parameters like the cropping box in the `model.pth` file since we always want to use the ones we trained with.

Reply:
I was wondering exactly the same. Looks like soem way to store the model parameters is important.
I think it's also important in order to make it easier to run training or inference without having a gazillion arguments to the script